.github/workflows/build-ros2-ws-action.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master]  
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        run: git clone -b feature/fix-using-grpc https://github.com/ichiro-its/aruku.git
+
+      - name: Checkout Kansei
+        run: git clone https://github.com/ichiro-its/kansei.git                 
+
+      - name: Checkout Kansei Interfaces
+        run: git clone https://github.com/ichiro-its/kansei_interfaces.git         
+      
+      - name: Checkout Aruku Interfaces
+        run: git clone https://github.com/ichiro-its/aruku_interfaces.git     
+
+      - name: Checkout Keisan
+        run: git clone https://github.com/ichiro-its/keisan.git        
+
+      - name: Checkout tachimawari
+        run: git clone https://github.com/ichiro-its/tachimawari.git  
+
+      - name: Checkout tachimawari interfaces
+        run: git clone https://github.com/ichiro-its/tachimawari_interfaces.git    
+
+      - name: Setup cmake        
+        run: sudo apt install -y build-essential autoconf libtool pkg-config cmake
+
+      - name: Setup abseil for grpc              
+        run: |
+          git clone https://github.com/abseil/abseil-cpp.git
+          cmake -S abseil-cpp -B abseil-cpp/build 
+          cmake --build abseil-cpp/build
+          cmake --install abseil-cpp/build
```

Comment:
Maybe we need super user access for installing this?

```suggestion
          sudo cmake --install abseil-cpp/build
```

<img width="602" alt="image" src="https://github.com/ichiro-its/aruku/assets/10202888/a1d3670b-a2e9-49ec-91e7-d2a13197f670">